### PR TITLE
Finished hw01 Przezdziecki

### PR DESCRIPTION
**Pull request by AdrianDP1**
The assignment was fairly easy, as Markdown syntax tries to be quick to learn and simple to use based on the documentation. Included in the Markdown is my name and a list including my major, other projects I've worked on, as well as an image - all edited in R Studio.

The one thing I had to look up was if I had to use quotation marks around the URL to my image or not.

I didn't follow the video completely, as I have GitHub Desktop on my computer, and that simplifies the copying, committing, and pushing to the repository.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,16 @@
...
+# hw01
+
+Hello, my name is Adrian Przezdziecki, a rising 4th-year in your *Computing for the Social Sciences* class.
+
+## General Info
+
+* My major is **Public Policy Studies** in the College.
+  * I initially started out as a Physics major before getting into the wonderful world of Social   Science.
+* I actually have experience with R in the geographical science world, particularly working with the *sf* package
+  * [Here](https://github.com/JJiang11/flightpath) is a package a friend and I created for our GIS class that uses an API and various packages to analyze real-time flight paths.
```
Comment (wanitchayap) [inline README.md:10]: That is really cool!

File: README.md [hunk 2]
```diff
@@ -1 +1,16 @@
...
+  * Granted, I already learned something new about R with the ggplot package reading, as I would just use general plot() functions in R before.
```
Comment (wanitchayap) [inline README.md:11]: Glad to here that. I think `ggplot` is really useful :) Still, I think my favorite things about R beyond base R are `tidyverse` and piping.

Comment (wanitchayap) [reply README.md:11]: Just to make it cooler, you can make your "plot()" here become code text: `plot()`

**Review by wanitchayap — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Improvement |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |      X     |              |            |
| **Coding strategy**         |     X      |              |            |
| **Presentation**            |           |       X       |            |
| **Achievement, creativity** |     X      |              |            |
| **Reproducibility**         |     X      |              |            |

Remarks:
* Great job! 
* Frequent commits 👍 
* More than 4 Markdown aspects included

File: README.md [hunk 3]
```diff
@@ -1 +1,16 @@
-# hw01
\ No newline at end of file
+# hw01
+
+Hello, my name is Adrian Przezdziecki, a rising 4th-year in your *Computing for the Social Sciences* class.
+
+## General Info
```
Comment (klyzhng) [inline README.md:5]: Good use of different header types!

File: README.md (hunk 1 above)
Comment (klyzhng) [inline README.md:10]: Wow! I actually like tracking flights for fun so this is AWESOME!!! I will try out you and your friend's package sometime when I have a better grasp of R. Also, nice use of nested bullet points!

File: README.md [hunk 4]
```diff
@@ -1 +1,16 @@
...
+* My major is **Public Policy Studies** in the College.
+  * I initially started out as a Physics major before getting into the wonderful world of Social   Science.
+* I actually have experience with R in the geographical science world, particularly working with the *sf* package
+  * [Here](https://github.com/JJiang11/flightpath) is a package a friend and I created for our GIS class that uses an API and various packages to analyze real-time flight paths.
+  * Granted, I already learned something new about R with the ggplot package reading, as I would just use general plot() functions in R before.
+ 
+## Image
+Here's a picture of me so as to include additional Markdown elements:
+
+![*Hey look, it's a blurry picture of me!*](https://github.com/AdrianDP1/hw01/blob/master/me.jpg?raw=true)
```
Comment (klyzhng) [inline README.md:16]: nice image embedding!

File: README.md [hunk 5]
```diff
@@ -1 +1,16 @@
-# hw01
\ No newline at end of file
+# hw01
+
```
Comment (klyzhng) [inline README.md:2]: Thanks for sharing about your use of Github Desktop -- I'll try it sometime!

**Pull request closed without merging**